src/data/reducers/ProfileReducer.js

```diff
@@ -0,0 +1,39 @@
+import { SAVE_USER_PROFILE } from '../../actions/profileActions';
```

Comment:
Can you remove the related reducer from `frontend-auth`?

Reply:
Sure can.